compose-v2/galaxy-slurm/Dockerfile

```diff
@@ -0,0 +1,55 @@
+ARG SETUP_REPO=andreassko
+ARG IMAGE_TAG=latest
+
+FROM buildpack-deps:18.04 as galaxy_dependencies
+
+ARG GALAXY_RELEASE=release_19.09
```

Comment:
this one should also be upgraded ...

Reply:
Ouh, good point. Thanks, I will fix that :)